frontend/src/sections/WhatIsDjango/WhatIsDjango.tsx

```diff
@@ -3,6 +3,7 @@
 import Image from 'next/image'
 import { Button } from '@components'
 import useWidth from '@/hooks/useWidth'
+import DataCard from '@/components/DataCard/datacard'
```

Comment:
import from barrel

Reply:
unclear elaborate please

Reply:
We need to export DataCard from the index.ts file in `frontend/src/components`.  This will eliminate the need of this line and we can import DataCard along with Button in line 4 like 
`import { Button , DataCard } from '@components'`

Reply:
Tried with index.ts like other components, but continues to error.

Reply:
- Change this line `function DataCard() {` to `export const  DataCard = () => {`
- Remove this line `export default DataCard` from end of file.
- create `index.ts` in `components/DataCard` and add `export * from './datacard'`
- add `export * from './DataCard'` in `components/index.ts`
- try importing DataCard in same line as Button